web-search-engines/langchain4j-web-search-engine-searchapi/src/test/java/SearchApiGoogleSearchEngineIT.java

```diff
@@ -0,0 +1,85 @@
+import dev.langchain4j.web.search.WebSearchEngine;
+import dev.langchain4j.web.search.WebSearchOrganicResult;
+import dev.langchain4j.web.search.WebSearchRequest;
+import dev.langchain4j.web.search.WebSearchResults;
+import dev.langchain4j.web.search.searchapi.SearchApiWebSearchEngine;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@EnabledIfEnvironmentVariable(named = "SEARCHAPI_API_KEY", matches = ".+")
+class SearchApiGoogleSearchEngineIT {
+
+    @Test
+    void should_search() {
+        // given
+        SearchApiWebSearchEngine searchEngine = SearchApiWebSearchEngine.builder()
+                .apiKey(System.getenv("SEARCHAPI_API_KEY"))
+                .build();
+
+        // when
+        WebSearchRequest request = WebSearchRequest.builder()
+                .searchTerms("What is Langchain4j?")
+                .maxResults(5)
+                .build();
+        WebSearchResults webSearchResults = searchEngine.search(request);
+
+        // then
+        List<WebSearchOrganicResult> results = webSearchResults.results();
+        assertThat(results).hasSize(5);
```

Comment:
This assert fails. Obviously the suggestion https://github.com/langchain4j/langchain4j/issues/1132#issuecomment-2144059116 is not implemented.
OMG - there are two concurrent PRs for this feature 😄 I've never experienced that 😄